pgjdbc/src/main/java/org/postgresql/jdbc/TypeInfoCache.java

```diff
@@ -286,7 +286,22 @@ private PreparedStatement prepareGetTypeInfoStatement() throws SQLException {
   }
 
   public synchronized int getSQLType(String pgTypeName) throws SQLException {
-    return getSQLType(castNonNull(getPGType(pgTypeName)));
+    /*
+    Get a few things out of the way such as arrays and known types
+     */
+    if (pgTypeName.endsWith("[]")) {
+      return Types.ARRAY;
+    } else {
```

Comment:
this does not need to be in an else block (the if above always returns)

Reply:
I was going to say it reads better, but I don't care that much.